jupyter_scheduler/models.py

```diff
@@ -193,9 +189,10 @@ class UpdateJobDefinition(BaseModel):
     name: Optional[str] = None
     url: Optional[str] = None
     schedule: Optional[str] = None
-    timezone: Optional[str] = None  # Should be a timezone e.g., US/Eastern, Asia/Kolkata
+    timezone: Optional[str] = None
```

Comment:
I like the idea of adding that this should be in tzdata format (although `US/Eastern` is an alias for `America/New_York`).

Reply:
I will add those in the docs; haven't figured out the mechanism to generate them yet.